features.yml.dist

```diff
@@ -7,7 +7,3 @@ toggles:
     name: jobs-endpoint
     conditions: {}
     status: inactive
-  import-from-sapi:
```

Comment:
It seems like this file https://github.com/cultuurnet/appconfig/blob/main/files/udb3/production/udb3-silex/features.yml could also use a cleanup.

Reply:
Yes, after the next chained PR is deployed we can remove the features.yml for every environment

Reply:
Oh I missed this and commented the same on your other PR about feature toggles 😄